composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "laravel/framework": "5.1.*||5.2.*",
-        "nodes/dingo": "^1.0",
+        "dingo/api": "dev-master#8cd1a6cd7d2ba9fc96841fd0745ef8fc1ce2a530",
```

Comment:
We should never lock the package to a specific commit hash. That's why we forked the repository in the first place. Instead we need to update the forked repository :)